src/Client.js

```diff
@@ -2180,6 +2180,18 @@ class Client extends EventEmitter {
             return await window.Store.AddressbookContactUtils.deleteContactAction(phoneNumber);
         }, phoneNumber);
     }
+
+    /**
+     * Get lid or phone number 
+     * @param {string} userId
+     * @returns {Promise<string>}
+     */
+    async getContactLidOrPhone(userId) {
+        return await this.pupPage.evaluate(async (userId) => {
+            const wId = await window.Store.WidFactory.createWid(userId);
+            return userId.indexOf('lid')>-1 ? await window.Store.LidUtils.getPhoneNumber(wId)._serialized : await window.Store.LidUtils.getCurrentLid(wId)._serialized;
```

Comment:
What do you think about simplifying here?

```suggestion
            return wId.server === 'lid'
                ? await window.Store.LidUtils.getPhoneNumber(wId)._serialized
                : await window.Store.LidUtils.getCurrentLid(wId)._serialized;
```